docs/apis/streaming-writer-api/create-stream.md

```diff
@@ -12,12 +12,9 @@ supplying any other additional properties required.
 
 ## Before you begin
 
-  - You should have a [Workspace set
-    up](../../platform/definitions.md#_workspace) with at least [one
-    Topic](../../platform/definitions.md#_topics).
+  - You should have a [Workspace set up](../../../platform/definitions.html#workspace) with at least one [Topic](../../../platform/definitions.html#topics).
```

Comment:
Should be `../../platform/definitions.html#workspace` and `../../platform/definitions.html#topics`.